types/react/test/managedAttributes.tsx

```diff
@@ -182,27 +115,28 @@ interface LeaveMeAloneDtslint { foo: string; }
 // const memoTests = [
 //     // $ExpectError
 //     <MemoFunctionalComponent />,
-//     // $ExpectError won't work as long as FunctionalComponent doesn't work either
+//     // Requires TypeScript >= 3.2; see comment re FunctionalComponent above
 //     <MemoFunctionalComponent str='abc' />,
 //     // $ExpectError
 //     <MemoAnnotatedDefaultProps />,
 //     <AnnotatedDefaultProps str='abc' />,
-//     // $ExpectError this doesn't work despite JSX.LibraryManagedAttributes returning the correct type
+//     // Requires TypeScript >= 3.2; see comment re FunctionalComponent above
```

Comment:
Oh, cool, they actually work :tada:

The only way I had to check when I was writing these was using `JSX.LibraryManagedAttributes` directly and seeing if the types looked right.